samples/test/steps/then.ts

```diff
@@ -19,12 +20,22 @@ Then(
   /^she should see a message on the Login form "(?<message>.+?)"$/,
   checkFormMessage
   );
-  
+
 Then(
   /^the Root Page shows links to the Entry Points$/,
   checkGuest
 );
 
+Then(
+  /table with the claims from the \/userinfo response$/,
+  checkProfile
+);
+
+Then(
+  /Mary sees a (.*) button$/,
+  isVisible
```

Comment:
let's make this step less generic (e.g. checkLogoutButton/checkButton)
also, we should probably avoid specifying actor name in the regexp

Reply:
Fixed